digitiser-aggregator/src/main.rs

```diff
@@ -81,6 +88,8 @@ struct Cli {
     otel_level: LevelFilter,
 }
 
+type ChannelSender = Sender<AggregatedFrame<EventData>>;
```

Comment:
Could this type have a more descriptive name? `ChannelSender` is rather abstract.